### DISCUSSION TITLE
How to run the demos and connect to GoPro Hero 11?

### DISCUSSION THREAD

**Discussion by djpirra** [gopro/OpenGoPro#294] [General]
Hi,

I have tried to run the webcam demo on my Windows machine but I am not being able to connect.

This is the error:
GoProNotOpened: GoPro is not correctly open: Serial / IP has not yet been discovered

I have tried while connected to regular Wifi and also connected to the GoPro Wifi without success.

The objective is to start a development of a OpenCV object detection.

Any help on what might be failing here?

**Comment by tcamise-gpsw** [comment]
Which demo are you referring to?
Also please post the entire log.

**Comment by djpirra** [comment]
The demo for the webcam and how to get the log? You mean the log file
generated? There is not much information there.


On Thu, 12 Jan 2023 at 21:58 Tim Camise ***@***.***> wrote:

> Which demo are you referring to?
> Also please post the entire log.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/OpenGoPro/discussions/294#discussioncomment-4670308>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADZ5Q5UMUJISSJLHBLTKBU3WSBA4JANCNFSM6AAAAAATY6AGA4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Melhores Cumprimentos,
Luis Simões
